tests/account.rs

```diff
@@ -17,8 +16,8 @@ fn create_and_delete_account() {
         earning: None,
         is_available: false,
     };
-    let account = account::create_account(&conn, user_id, new_account).unwrap();
-    account::delete_account(&conn, &account.id, &user_id).unwrap();
+    let account = conn.create_account(user_id, new_account).unwrap();
+    conn.delete_account(&account.id, &user_id).unwrap();
```

Comment:
Replacing unwrap by expect can make this more clear.

Reply:
Fixed.